pyvrp/cpp/search/Route.h

```diff
@@ -9,6 +9,17 @@
 
 namespace pyvrp::search
 {
+/**
+ * This ``Route`` class supports fast delta cost computations and in-place
+ * modification. It can be used to implement move evaluations.
+ *
```

Comment:
We should probably add here that the in-place modifications do not automatically update the Route data, which should be done by explicitly calling `update()`.